src/autosuggest/analytics-metadata/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { GeneratedAnalyticsMetadataSelectableItemSelect } from '../../internal/components/selectable-item/analytics-metadata/interfaces';
+
+export type GeneratedAnalyticsMetadataAutosuggestSelect = GeneratedAnalyticsMetadataSelectableItemSelect;
+
+export interface GeneratedAnalyticsMetadataAutosuggestClearInput {
+  action: 'clearInput';
+  detail: {
+    label: string;
+  };
+}
```

Comment:
It is not, I am still adding all of them in the component analytics interface, so that I will be able to automatically generate documentation in the future. I did push an improvement to at least use it in the input component, and import it from there.